### PR TITLE
Code style cleanup for App.kt

### PR DESCRIPTION
**Pull request by quh4gko8**

### CODE REVIEW

File: app/src/main/java/org/grapheneos/apps/client/App.kt [hunk 1]
```diff
@@ -211,12 +211,7 @@ class App : Application() {
 
         if (!isServiceRunning) {
             if (foregroundServiceNeeded && !isSeamlessUpdateRunning()) {
-                startService(
-                    Intent(
-                        this@App,
-                        KeepAppActive::class.java
-                    )
-                )
+                startService(Intent(this@App, KeepAppActive::class.java))
```
Comment (Zanthed) [inline app/src/main/java/org/grapheneos/apps/client/App.kt:214]: I don't know the Kotlin official codestyle off the top of my head, but if this is going against it then it shouldn't be changed here.

Comment (quh4gko8) [reply app/src/main/java/org/grapheneos/apps/client/App.kt:214]: It should only apply for code exceeding 100 columns, and this one is nowhere near 100, and is not being warned in Android Studio

Comment (quh4gko8) [reply app/src/main/java/org/grapheneos/apps/client/App.kt:214]: For example: https://github.com/GrapheneOS/Apps/blob/a58545e031ff00b4b3184aacab6e433a3e9afba7/app/src/main/java/org/grapheneos/apps/client/App.kt#L1012

https://github.com/GrapheneOS/Apps/blob/a58545e031ff00b4b3184aacab6e433a3e9afba7/app/src/main/java/org/grapheneos/apps/client/App.kt#L214

Comment (Zanthed) [reply app/src/main/java/org/grapheneos/apps/client/App.kt:214]: Ah okay, was just checking. My eyes couldn't tell if it was over 100 or not lol.

**Pull request merged**